packages/website/docs/autocomplete-layout-classic.md

```diff
@@ -0,0 +1,210 @@
+---
+id: autocomplete-layout-classic
+---
+
+The Classic layout provides components for Autocomplete experiences.
+
+This layout relies on the [Autocomplete Classic Theme](autocomplete-theme-classic). Make sure to install it as well.
+
+## Installation
+
+First, you need to install the layout.
+
+```bash
+yarn add @algolia/autocomplete-layout-classic@alpha
+# or
+npm install @algolia/autocomplete-layout-classic@alpha
+```
+
+Then import it in your project:
+
+```js
+import {
+  NavigationCommands,
+  SearchByAlgolia,
+} from '@algolia/autocomplete-layout-classic';
+```
+
+If you don't use a package manager, you can use a standalone endpoint:
+
+```html
+<script src="https://cdn.jsdelivr.net/npm/@algolia/autocomplete-layout-classic@alpha"></script>
+```
+
+## Examples
+
+With default translations:
+
+```tsx
+/** @jsx h */
+import { autocomplete } from '@algolia/autocomplete-js';
+import {
+  NavigationCommands,
+  SearchByAlgolia,
+} from '@algolia/autocomplete-layout-classic';
+import { h, render } from 'preact';
+
+import '@algolia/autocomplete-theme-classic';
+
+autocomplete({
+  // ...
+  components: {
+    NavigationCommands,
+    SearchByAlgolia,
+  },
+  render({ sections, Fragment, components }, root) {
+    render(
+      <Fragment>
+        <div className="aa-PanelLayout aa-Panel--scrollable">{sections}</div>
+        <footer className="aa-PanelFooter">
+          <components.NavigationCommands />
+          <components.SearchByAlgolia />
+        </footer>
+      </Fragment>,
+      root
+    );
+  },
+});
+```
+
+With French translations:
+
+```tsx
+/** @jsx h */
+import { autocomplete } from '@algolia/autocomplete-js';
+import {
+  NavigationCommands,
+  SearchByAlgolia,
+} from '@algolia/autocomplete-layout-classic';
+import { h, render } from 'preact';
+
+import '@algolia/autocomplete-theme-classic';
+
+autocomplete({
+  // ...
+  components: {
+    NavigationCommands,
+    SearchByAlgolia,
+  },
+  render({ sections, Fragment, components }, root) {
+    render(
+      <Fragment>
+        <div className="aa-PanelLayout aa-Panel--scrollable">{sections}</div>
+        <footer className="aa-PanelFooter">
+          <components.NavigationCommands
```

Comment:
could you in this case not use it directly, without passing to `components`?

Reply:
my linter doesn't detect that `components` variable is used in the code. I guess because it struggles to detect the html tag and make the relation? but it's not great.